www/_posts/2020-06-13-plugins-release.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+author:
+    name: Niklas Merz
+title:  "Splashscreen and InAppBrowser plugin Released!"
+categories: news
+tags: release plugins
+---
+
+We are happy to announce that we have just released an update to some plugins!
+
+* [cordova-plugin-inappbrowser@4.0.0](https://www.npmjs.org/package/cordova-plugin-inappbrowser)
+* [cordova-plugin-splashscreen@5.0.4](https://www.npmjs.org/package/cordova-plugin-splashscreen)
+
+## Release Highlights
+
+### InAppBrowser
+
+### cordova-common
+
+Cordova-iOS Platform 6.x will have splash screen integrated and no longer needs the plugin. This releas prevents the plugin from being added to the iOS 6.x platform.
```

Comment:
```suggestion
In the Cordova-iOS 6.x platform, the splash screen feature has been integrated into its core and no longer needs this plugin. This release prevents the plugin from being added to the iOS 6.x platform.
```

Reply:
FYI trying to use it with cordova-ios looks like this:

```
Installing "cordova-plugin-splashscreen" for ios
Plugin doesn't support this project's cordova-ios version. cordova-ios: 6.0.0, failed version requirement: <6.0.0
Skipping 'cordova-plugin-splashscreen' for ios
Adding cordova-plugin-splashscreen to package.json
```